<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>House_plant_guide</class>
 <widget class="QWidget" name="House_plant_guide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>House_plant_guide</string>
  </property>
  <widget class="QPushButton" name="button_enter_to_guide">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(201, 160, 220);
}
</string>
   </property>
   <property name="text">
    <string>Каталог растений</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_to_admin">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>490</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color: rgb(201, 160, 220);
}
</string>
   </property>
   <property name="text">
    <string>Войти в аккаунт администратора</string>
   </property>
  </widget>
  <widget class="QPushButton" name="my_plants">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(201, 160, 220);
}
</string>
   </property>
   <property name="text">
    <string>Мои растения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
